README.md

```diff
@@ -260,6 +279,58 @@ exit prompt.
 2021/06/11 15:00:19 [info] success.
 ```
 
+## Usage: ECS scheduled tasks with Fargate
+
+As a usecase, Consider using ECS scheduled tasks.
+
+example task definition
+```json
+{
+    "taskDefinition": {
```

Comment:
`aws ecs register-task-definition --cli-input-json`  requires JSON file which not having taskDefinition element.
See also https://docs.aws.amazon.com/cli/latest/reference/ecs/register-task-definition.html#examples